pages/index.js

```diff
@@ -66,15 +74,11 @@ const index = function({ size }) {
           <ScrollElement name="team">
             <Team />
           </ScrollElement>
-
-          <ScrollElement name="contact" className="contact">
-            <Contact />
```

Comment:
I believe the conclusion was we were going to keep the contact form and switch it for a "success" state when it submits. It wouldn't transition into the mail client though, just say "success" and submit the form to Hubspot.
By preserving the contact form, even if the user has dismissed the HS form, they can submit the info to us.